### PR TITLE
GovCloud ARN is different of standard one

### PR DESCRIPTION
**Pull request by Twsouza**
The standard ARN does not work on AWS GovCloud, it has to use the `arn:aws-us-gov`.

### CODE REVIEW

**Review by ddoc — COMMENTED** [review]
was formation-govcloud.json not in the repo already? the documentation changes look ok

**Comment by Twsouza** [comment]
> was formation-govcloud.json not in the repo already? the documentation changes look ok

No, I tried to install it and end up with an error when we tried to create the stack.

**Comment by heronrs** [comment]
Thanks Taynan. Looks good to me. Maybe a single formation file would be better but it can get clumsy given the cf syntax

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ You can do this easily via your AWS Web Console, uploading the `formation.json`
 
 If you are using the AWS GovCloud, you have to run this formation stack:
```
[the remark this answers is not in the document]
Comment (Twsouza) [reply README.md]: @ddoc done!

**Review by heronrs — APPROVED** [review]

**Pull request merged**